packages/carousel/src/Carousel.js

```diff
@@ -43,13 +43,15 @@ export const Carousel = ({
   visibleSlides = 1,
   arrowsPosition = 'side',
   slideSpacing = 2,
-  buttonColorProps,
   onSlideChange,
+  sideButtonMargin = '-30px',
```

Comment:
If we made `buttonSize` a `number` type, we could set sideButtonMargin = `'${buttonSize / 2}px'`